ui/pages/Staff/constants.js

```diff
@@ -90,6 +92,16 @@ const formatFamilySummary = row =>
     }
   </div>
 
+const formatIDLink = row =>
+  <div>
+    <Link to={`/project/${row.project_guid}/family_page/${row.family_guid}`} target="_blank">{row.family_id}</Link>
+  </div>
+
+const formatSuccessStoryTypes = row =>
+  row.success_story_types && <div>{row.success_story_types.map(tag => <div><small>{FAMILY_SUCCESS_STORY_TYPE_OPTIONS_LOOKUP[tag].name}</small></div>)}</div>
+
+const formatDiscoveryTags = row => <div>{row.extras_variant_tag_list}</div>
```

Comment:
If all you are doing is grabbing a different value from the object to display, why not just make `extras_variant_tag_list` the name and get rid of the format function?